src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1592,6 +1594,9 @@ namespace NServiceBus.Config
             "s => settings.NumberOfRetries(5);. Will be removed in version 7.0.0.", true)]
         public int MaxRetries { get; set; }
     }
+    [System.ObsoleteAttribute("Use of the application configuration file to configure NServiceBus is discouraged" +
```

Comment:
hm I don't like UnicastBus too much, it has no corresponding feature as "auditing". Maybe just "the endpoint"?